src/UIKit/TablePagination.vue

```diff
@@ -34,10 +34,15 @@
 import { ROWS_PER_PAGE } from "@/constants";
 import { PaginationCallType } from "@/types";
 import { defineComponent, ref } from "vue";
+import { LeftOutlined, RightOutlined } from "@ant-design/icons-vue";
```

Comment:
Imported but using them???

Reply:
Wasn't able to see the next, prev icons in pagination, when debugged I found the issue, so fixed it
LeftOutlined -> <left-outlined/>
RightOutlined -> <right-outlined/>